internal/export/distro/format_test.go

```diff
@@ -38,7 +38,8 @@ func TestJson(t *testing.T) {
 	if err := json.Unmarshal(out, &eventOut); err != nil {
 		t.Fatalf("Error unmarshalling event: %v", err)
 	}
-	if !reflect.DeepEqual(eventIn, eventOut) {
+	//if !reflect.DeepEqual(eventIn, eventOut) {
```

Comment:
Why this commented code?

Reply:
I'll remove the comment. I replaced it with the comparison on the line underneath.
`if eventIn.String() != eventOut.String()`
For more information, see the second bullet point in the following comment https://github.com/edgexfoundry/go-mod-core-contracts/issues/66#issuecomment-482732387